### DISCUSSION TITLE
Camera-Based Motion Detection?

### DISCUSSION THREAD

**Discussion by cchance27** [blakeblackshear/frigate#11865] [General]
Sorry if was asked i tried searching but searching for motion detection and camera based... ya a billion responses 0 helpful.

Just wondering if theirs anyway to use the camera based motion detection to trigger frigate, i've got a pretty weak cpu but a tpu for object recognition, feel like it would be great if we could use the camera based motion detection for triggering motion actions. I know scrypted does it for stuff like hikvision, if frigate could use the camera based motion as the initial trigger i feel like it would open up even lighter hardware requirements (depending on camera support)

**Comment by NickM-27** [comment]
there is not, the camera motion detection does not provide enough information for frigate to do its object detection accurately and often is not fast enough.

It would also be a lot more work to support each camera's proprietary API (if one exists at all)

**Reply by cchance27** [reply]
Thats sad i'd have hoped something like that would allow less powerful cpu's with a tpu attached to rely on the initial motion trigger to be camera based as many of them have various versions of webhook or other calls when motion is triggered

**Reply by NickM-27** [reply]
every implementation I have seen, even onvif, does not provide enough information to be usable by frigate. And the latency would cause issues as well. We have other ideas to reduce the CPU usage of motion detection but even now the hardware required is fairly minimal
